rotkehlchen/chain/evm/decoding/decoder.py

```diff
@@ -113,18 +114,21 @@
 MIN_LOGS_PROCESSED_TO_SLEEP = 1000
 
 
-def decode_safely(
+def safely_apply_decoding_fn(
         msg_aggregator: 'MessagesAggregator',
-        tx_hash: EVMTxHash,
         chain_id: ChainID,
         func: Callable,
+        tx_hash: EVMTxHash | None = None,
         *args: tuple[Any],
         **kwargs: Any,
 ) -> tuple[Any, bool]:
     """
     Wrapper for methods that execute logic from decoders. It handles all known errors
     by logging them and optionally sending them to the user.
 
+    tx_hash is used when a single transaction gets decoded and is used only to log more
+    information in case of error.
```

Comment:
```suggestion
    tx_hash is used only to log more information in case of error when decoding a single transaction.
```
something like this is a bit more concise and doesn't repeat the word `used`